frontend/src/redux/reducers/userAuthReducer.tsx

```diff
@@ -2,13 +2,13 @@ import {
   UserAuthAction,
   UserAuthActionTypes
 } from 'redux/action-types/userAuthActionTypes';
-import { UserLoginState } from 'redux/ts-types/userAuth';
+import { UserInfo, UserLoginState } from 'redux/ts-types/userAuth';
 
 const initialState: UserLoginState = {
   loading: false,
   error: null,
   isLogged: false,
-  userInfo: {}
+  userInfo: {} as UserInfo
```

Comment:
you should change this after PR O. Mazur

Reply:
I discussed this with Sasha. I couldn't finish my task because of that. That's why Sasha has to give a review.